docs/reference/token-scopes.md

```diff
@@ -0,0 +1,38 @@
+# Token scopes
+
+## Fine grained access token scopes
+
+### Organizational Runners
+
+The following are the permissions scopes required for the GitHub runners when registering as an
+organisational runner.
+
+Organisation:
+
+- Self-hosted runners: read & write
+
+Repository:
+
+- Administration: read
+- Contents: read
+- Pull requests: read
+
+### Repository Runners
+
+The following are the permissions scopes required for the GitHub runners when registering as an
+repository runner.
+
+- Contents: read
+- Metadata: read
+- Pull requests: read
+
+## Personal access token scope
+
+To use this charm for GitHub repositories, the following scopes should be selected:
+
+- `repo`
+
+To use this charm for GitHub organisations, the following scopes should be selected:
+
+- `repo`
+- `admin:org`
```

Comment:
Maybe also use `##  Organizational runners` and `## Repository runners` headers with similar wording, to make it look more uniform.